dataland-frontend/tests/e2e/fixtures/CompanyFixtures.ts

```diff
@@ -1,4 +1,4 @@
-import { faker } from "@faker-js/faker/locale/de";
```

Comment:
Is this only the german localization?

Reply:
I changed many of the occurences of a german faker to the english one in a different branch already. However, through some merge conflict or something this one returned. And got fixed here again.